assets/js/components/surveys/SurveyQuestionOpenText.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * SurveyQuestionOpenText component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import SurveyHeader from './SurveyHeader';
+import { TextField, Input, HelperText } from '../../material-components';
+import Button from '../Button';
+
+const MAXIMUM_CHARACTER_LIMIT = 100;
+
+const SurveyQuestionOpenText = ( {
+	question,
+	answerQuestion,
+	placeholder,
+	subtitle,
+	dismissSurvey,
+} ) => {
+	const [ value, setValue ] = useState( '' );
+
+	const handleSubmit = () => {
+		answerQuestion( { answer: value } );
+	};
+
+	const onChange = useCallback(
+		( event ) => {
+			setValue( event.target.value?.slice( 0, MAXIMUM_CHARACTER_LIMIT ) );
+		},
+		[ setValue ]
+	);
+
+	return (
+		<div className="googlesitekit-survey__open-text">
+			<SurveyHeader title={ question } dismissSurvey={ dismissSurvey } />
+			<div className="googlesitekit-survey__body">
+				<div>
+					<TextField
+						name="open-text"
+						helperText={ <HelperText>{ subtitle }</HelperText> }
+						onChange={ onChange }
+						label={ placeholder }
+						noLabel
```

Comment:
Why is there no label for this input? I don't see that mentioned in the ACs/IB, and doing this severely impacts accessibility of the input.

Reply:
To match the design. If I remove this prop then when entering text the placeholder (which is really the label) does this

![label](https://user-images.githubusercontent.com/8496063/128747454-974dec92-8c17-4743-9d99-bf225b544eee.png)

Reply:
Right, that is a quirk of material inputs.

Then let's add a manual `<label>` tag and connect the ID for this element and the label. We can't have inputs without labels or users of assistive technology won't be able to use the form.

Reply:
Sure thing. I'll do the same as I did on the other PRs :+1: